includes/utils.inc

```diff
@@ -525,6 +525,10 @@ function wf_crm_array2str($arr) {
  *   Result of API call
  */
 function wf_civicrm_api($entity, $operation, $params) {
+  if (is_null($entity)) {
+    return;
+  }
```

Comment:
This is just a weird situation. But since this function promises to return an array, I'd suggest:
```suggestion
  if (!$entity) {
    return [];
  }
```

Reply:
Turns out it is not just a weird situation but also an intermittent situation :-) - First reported last Fall by @jitendrapurohit - this may be a permanent avoid the situation and that is a good solution 👍